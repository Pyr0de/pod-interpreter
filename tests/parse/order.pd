(10+10)*12;
(-100+23*12);
(((10)));
(!true);
(10*12-(12)*12) >= 10+15;
(10+23/12)+125*-235;
-12+-13*13;
