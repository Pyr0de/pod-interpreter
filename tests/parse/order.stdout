ExitCode: 0
(* (group (+ 10 10) null) 12)
(group (+ (- 100 null) (* 23 12)) null)
(group (group (group 10 null) null) null)
(group (! true null) null)
(>= (group (- (* 10 12) (* (group 12 null) 12)) null) (+ 10 15))
(+ (group (+ 10 (/ 23 12)) null) (* 125 (- 235 null)))
(+ (- 12 null) (* (- 13 null) 13))
